### ISSUE TITLE
Monitoring can be stopped before the job termination

### ISSUE THREAD

**Issue by stebenoist** [ystia/yorc#438]
# Bug Report

## Description

When a job is monitored, a scheduled action is registered and is executed periodically.
Each time interval, a new monitoring task is registered and executed. When the task is done, it's deleted.
Before registering the new monitoring task, the previous monitoring task status is checked:
 1. task exists
 2. task status not running or initial

If the previous task is still running, no new task is registered and the scheduling continues.
But it happens that before check1 and check2 the task is deleted and an error is returned "missing status" when retrieving the task status : this stops the scheduled action and the monitoring

## Expected behavior
Check the task still exists at check2 as the process is asynchronous
Continue the job monitoring until its end

## Steps to reproduce the issue
10 long job running (30min) in parallel

## Priority

High
